server/tests/services/phone.test.js

```diff
@@ -0,0 +1,93 @@
+import phoneService from '../../services/phone';
+import config from '../../../config/env';
+import { build, fixtures } from '../factories';
+
+describe('## Phone Service', () => {
+  describe('# Call', () => {
+    const INVALID_NUMBER = '+15005550001';
+    const INVALID_NUMBER_ERROR_CODE = 21217;
+
+    let phoneDevice;
+
+    before((done) => {
+      build('device', fixtures.phoneDevice({ contactInformation: '+15005550006' }))
+        .then((device) => {
+          phoneDevice = device;
+          done();
+        });
+    });
+
+    it('should successfully make a phone call', (done) => {
+      phoneService.makeCall(null, null, phoneDevice)
+        .then((callInfo) => {
+          expect(callInfo).to.exist;
+          expect(callInfo.status).to.equal('queued');
+          done();
+        });
+    });
+
+    it('should fail to call an invalid number', (done) => {
+      const invalidNumberDevice = phoneDevice;
+      invalidNumberDevice.contactInformation = INVALID_NUMBER;
+
+      phoneService.makeCall(null, null, invalidNumberDevice)
+        .catch((err) => {
+          expect(err.code).to.equal(INVALID_NUMBER_ERROR_CODE);
+          expect(err.message).to.equal(`${INVALID_NUMBER} does not appear to be a valid phone number`);
+          done();
+        });
+    });
+  });
+
+  describe('# SMS', () => {
+    const INVALID_NUMBER = '+15005550001';
+    const INVALID_NUMBER_ERROR_CODE = 21211;
+
+    const CANT_ROUTE_NUMBER = '+15005550002';
+    const CANT_ROUTE_NUMBER_ERROR_CODE = 21612;
+
+    let smsDevice;
+
+    before((done) => {
+      build('device', fixtures.smsDevice({ contactInformation: '+15005550006' }))
+        .then((device) => {
+          smsDevice = device;
+          done();
+        });
+    });
+
+    it('should successfully send a sms message', (done) => {
+      phoneService.sendMessage(null, null, smsDevice)
```

Comment:
Nope, it uses their [test credentials](https://www.twilio.com/docs/api/rest/test-credentials) to return specific responses based on the 'to' phone number